client/jetpack-cloud/sections/partner-portal/assign-license-form/style.scss

```diff
@@ -29,17 +30,68 @@
 
 .assign-license-form__top {
 	display: flex;
+	flex-wrap: nowrap;
 	justify-content: space-between;
-}
+	align-items: center;
+	margin-bottom: 1rem;
 
-.assign-license-form__top > div {
-	align-self: flex-end;
-	margin-bottom: 0;
+	@include breakpoint-deprecated( '>660px' ) {
+		flex-wrap: wrap;
+	}
+
+	@include break-medium {
+		flex-wrap: nowrap;
+	}
 }
 
-.assign-license-form__top {
-	margin-bottom: 1rem;
-	margin-top: -1.5rem;
+.assign-license-form__controls {
+	display: flex;
+	flex-direction: column-reverse;
+	align-items: center;
+	flex-grow: 1;
+	position: fixed;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	padding: 0.875rem 0.875rem 1.5rem;
+	background: white;
+	box-shadow: 0 -1px 2px rgba( 0, 0, 0, 0.12 );
+
+	.button {
+		width: 100%;
+		padding: 8px 24px;
+		font-size: 1rem;
+		white-space: nowrap;
+	}
+
+	@include breakpoint-deprecated( '>660px' ) {
+		flex-direction: row;
+		flex-wrap: nowrap;
+		position: static;
+		padding: 0;
+		background: transparent;
+		box-shadow: none;
+
+		.button {
+			flex-grow: 1;
+			width: auto;
+		}
+	}
```

Comment:
IMO having the two actions horizontal to each other where possible seems a bit cleaner. Should we perhaps borrow that from the other PR?

**"Heyde PR"**

![Screenshot 2022-02-23 at 16 46 49](https://user-images.githubusercontent.com/3846700/155354583-9b752b4c-d6a3-4569-ad49-4725ff8614b5.png)

**This PR**
![Screenshot 2022-02-23 at 16 46 40](https://user-images.githubusercontent.com/3846700/155354590-b7872f09-ffd6-46a8-a949-fcd98affc29a.png)